README.md

```diff
@@ -1,2 +1,55 @@
 # telesto-base
-Base image for telesto.ai models
+Base image for telesto.ai models.
+
+# Instructions
+
+`telesto-base` contains a pip-installable Python package and a Docker image, allowing you to
+easily package your models for telesto.ai competitions.
+
+## telesto-base module
+To install the module, you can simply use pip:
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git
+```
+If you would like to use the latest not yet released version, you can install the one in the 
+`develop` branch.
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git@develop
+```
+
+## The base image
+The base image contains the pre-installed `telesto-base` module. Your submissions will use this
+as a base, so you'll only have to worry about the algorithms and not the packaging. To use it
+locally, you can pull the image from Docker:
+```
+docker pull telestoai/model-api-base:stable
+```
+or
+```
+docker pull telestoai/model-api-base:latest
+```
+for the one in the `develop` branch.
+
+Alternatively, the image can also be built locally with the command 
+```
+docker build -t telestoai/model-api-base -f Dockerfile .
```

Comment:
This command will build and label an image with `:latest` tag, I think. It might matter if we recommend them to pull a `:stable` image from Docker Hub.

Reply:
Is there a way to set this in Docker? If not, than the `latest` and `stable` tags might not be a good idea. How to people usually do this?

Reply:
Yes, you can specify a tag in the image label `-t telestoai/model-api-base:stable`.
I think tags should work for us in this case, we just need to decide how to use them in a way that's less confusing for experts.

Reply:
What do you suggest?

Reply:
The Docker documentation defines the `latest` tag as the latest stable version of the image. So we can just stick with it without explicitly using or mentioning it, so no need for `stable` tag either.

Reply:
How should we set up the GitHub Action workflows? Pushes to master publish the image on Docker Hub with the tag `stable`, while pushes to develop publish the image with `latest`. How shall we change this?

Reply:
Since the `latest` image should contain stable code I wouldn't update it from the `develop` but rather from the `master` branch. For the develop branch changes, we might consider using the `dev` tag.

Reply:
Changed it in the GitHub Actions workflow, let me know if it is right!